### ISSUE TITLE
Is this going to be the official ossec container?

### ISSUE THREAD

**Issue by tkent** [ossec/ossec-docker#2]
Hello there! I'm the author/maintainer of the current de-facto OSSEC container (the Xetus OSS one that you referenced).

It's on our radar to update the image and perform some overdue maintenance in the next few weeks. If you're going to be maintaining an official ossec-docker container, we should discuss if it makes sense to retire ours and contribute here instead.

**Comment by ddpbsd** [comment]
Hi @tkent ! @atomicturtle did the work on this one, so he can make the final decision. I'd personally love to see the efforts combined to make 1 docker image better.

**Comment by atomicturtle** [comment]
Yeah absolutely, I'd love to collaborate on it! I'd like to get these into the docker official builds repository, expand the documentation, etc. We could absolutely use the help on maintaining these.

**Comment by atomicturtle** [comment]
@tkent would you like to join us on our slack channel? Send me an email to scott @ ossec.net for an invite.

**Comment by tkent** [comment]
@atomicturtle, @ddpbsd Thanks for the quick responses. 

I'll send an email and we'll chat on slack when we can connect. A high bandwidth conversation is probably going to be more productive :-).
